### ISSUE TITLE
Confused about which plugin to use

### ISSUE THREAD

**Issue by typhoon2099** [angulartics/angulartics#583]
I have an Angular 1.6 app using Angulartics to send pageviews to Google Analytics. Our client now wants us to use Google Tag Manager to manage the analytics code but I can't figure out how to configure it. Do I need both `angulartics-google-analytics` and `angulartics-google-tag-manager` installed to do this? So far I have only been able to get a single pageview when the app is loaded up, nothing gets sent on page transitions (via UI-Router).

**Comment by dougludlow** [comment]
@typhoon2099 I was able to get Google Tag Manager working following the instructions to [import a container](https://github.com/angulartics/angulartics-google-tag-manager#container-import-installation-recommended).

I didn't need the `angulartics-google-analytics` plugin and I didn't need the "Global site tag (gtag.js) - Google Analytics" snippet. I did add the Google Tag Manager snippets.

I'm seeing the ui-router route in the active pages in Google Analytics, so everything appears to be working correctly. Just thought I'd share.
